_config.yml

```diff
@@ -28,7 +28,11 @@ exclude: ["Gemfile",
           "scripts/",
           ".gitignore",
           ".sass-cache/",
-          ".idea/"]
+          ".idea/",
```

Comment:
This `.idea` file is not checked in. So, why should this be excluded?

Reply:
I will remove this one.